core/src/main/scala/kafka/utils/CoreUtils.scala

```diff
@@ -66,11 +66,11 @@ object CoreUtils {
       action
     } catch {
       case e: Throwable => logLevel match {
-        case Level.ERROR => logger.error(e.getMessage, e)
-        case Level.WARN => logger.warn(e.getMessage, e)
-        case Level.INFO => logger.info(e.getMessage, e)
-        case Level.DEBUG => logger.debug(e.getMessage, e)
-        case Level.TRACE => logger.trace(e.getMessage, e)
+        case Level.ERROR => logging.error(e.getMessage, e)
+        case Level.WARN => logging.warn(e.getMessage, e)
+        case Level.INFO => logging.info(e.getMessage, e)
+        case Level.DEBUG => logging.debug(e.getMessage, e)
+        case Level.TRACE => logging.trace(e.getMessage, e)
```

Comment:
Fixed a bug where we were not using the passed `logging`.

Reply:
do we want to add a test (perhaps for one of the functions that are using this utility) which could have caught this? Could be done as a separate JIRA.

Reply:
I vote for separate JIRA.

Reply:
separate JIRA is good. Could be a great newbie task.

Reply:
https://issues.apache.org/jira/browse/KAFKA-15610